Menu/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200&icon_names=menu" />
+</head>
+<body>
+  
+    <div class="buttonDiv">
+<button id="menu-button"> <span class="material-symbols-outlined">menu</span></button>
+    </div>
+
+<div class="menu-location">
+ <div id="menu" class="menu">
+ <ul>
+<li> <a href="#" id="Letters" > Letters </a> </li>
+<li> <a href="#" id="Numbers" > Numbers </a></li>
+ </ul>
+</div>
+</div>
+
+
+
+<div id="content">
+    <dvi id="letters-page" class="page-content">
```

Comment:
> Starts with letters content shown and numbers hidden.

Add the `active` class here so that when the page is first loaded, the letters are there first.

```suggestion
    <div id="letters-page" class="page-content active" >
```